python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos_exploitation.py

```diff
@@ -70,27 +139,30 @@
 
 # %%
 # Get the multivariate basis
-# as a collection of Function
+# as a collection of :class:`~openturns.Function`.
 reduced = result.getReducedBasis()
 
 # %%
-# Get the orthogonal basis
+# Get the orthogonal basis.
 orthgBasis = result.getOrthogonalBasis()
 
 # %%
-# Get the distribution of variables Z
+# Get the standard distribution :math:`\mu` of the standardized
+# random vector :math:`\vect{Z}`.
 orthgBasis.getMeasure()
 
 # %%
-# Get the composed meta model which is the model of the reduced variables Z
-# within the reduced polynomials basis
+# Get the composed meta model :math:`\hat{h}` which is
+# the model of the standardized random vector :math:`\vect{Z}`
+# within the reduced polynomials basis.
 result.getComposedMetaModel()
 
 # %%
-# Get the meta model which is the composed meta model combined with the
-# iso probabilistic transformation
+# Get the meta model :math:`\hat{g}` which is the composed meta model composed with the
+# iso probabilistic transformation :math:`T`.
 result.getMetaModel()
 
 # %%
-# Get the projection strategy
+# Get the projection strategy: this is the method to estimate
+# the coefficients, i.e. regression or integration.
 algo.getProjectionStrategy()
```

Comment:
```suggestion
print(algo.getProjectionStrategy())
```
This is outside the scope of this PR, but these prints of complex objects are weird. Maybe we should try to do something with them rather than just print them? Like apply the metamodel and perform validation?

Reply:
This one is not-so-pretty:

![image](https://user-images.githubusercontent.com/31351465/236950269-758d77af-4037-4a31-8f96-b2602ac15836.png)

The repeated values in the `Point` are making the HTML crazy. Perhaps we can improve the `Point` pretty-print, so that when a repeated value is met, then something like `[0.0333] * 30` is printed?

Reply:
Unless #2282 does something about this, I think it would be better to apply my suggestion in this instance.

Reply:
No, #2282 will not change this. But another PR may do.

Reply:
So if I understand you correctly,you want to keep the code as is and improve the underlying pretty-print down the road?